kernel/aster-nix/src/process/signal/sig_queues.rs

```diff
@@ -142,6 +142,26 @@ impl SigQueues {
         let idx = (signum.as_u8() - MIN_RT_SIG_NUM) as usize;
         &mut self.rt_queues[idx]
     }
+    
+    pub fn sig_pending(&mut self) -> SigSet {
+        let mut pending = SigSet::new_empty();
+
+        // Process standard signal queues
+        for (idx, signal) in self.std_queues.iter().enumerate(){
+            if signal.is_some(){
+                pending.add_signal(SigNum::try_from(idx as u8 + MIN_STD_SIG_NUM).unwrap());
```

Comment:
Just use `SigNum::from_u8`, instead of `try_from`. Since it will never fail.

Reply:
FIXED